odxtools/element.py

```diff
@@ -0,0 +1,56 @@
+from dataclasses import dataclass
+from typing import List, Optional, TypedDict, cast
+from xml.etree import ElementTree
+
+from odxtools.exceptions import odxrequire
+from odxtools.utils import create_description_from_et
+
+from .odxlink import OdxDocFragment, OdxLinkId
+
+
+class BaseElementKwargs(TypedDict):
+    short_name: str
+    long_name: Optional[str]
+    description: Optional[str]
+
+
+@dataclass
+class NamedElement:
+    short_name: str
+    long_name: Optional[str]
+    description: Optional[str]
+
+    @staticmethod
+    def get_kwargs(
```

Comment:
The name is a bit too generic for my taste. Also, it should IMO not be a public method. What do you think about
```suggestion
    def __attribdict_from_et__(
```

Reply:
hm, okay. so far I assumed that `__foo__` was meant not to be called outside if the class or its descendants. (basically because of `__init__()` and `__post_init__()`). If we were not to replace that pattern anyway, I guess that we should rename `__init_from_et__()` to `__init_from_et()`?

Reply:
`__init_from_et` or `_init_from_et` depending if other files may call it or not

Reply:
agreed. Anyway, that's stuff for another day...